modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/Search.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayButtonWithIcon} from '@clayui/button';
+import {ClayInput} from '@clayui/form';
+import React from 'react';
+
+interface SearchInterface {
+	onSearch: Function;
+	query: string;
+}
+
+const Search = ({onSearch, query}: SearchInterface) => (
```

Comment:
I may be wrong in this, it has been a while, but I think this won't be properly read in the devtools when the code is mangled. Can we make this a named function instead?

```js
function Search(
```

Reply:
Why this component specifically @bryceosterhaus?

Reply:
I meant to say that for all components actually, this was just the one I noticed it on, I probably missed other instances of it.

I thought we used to have an eslint rule for this exact thing... [rule is here](https://github.com/liferay/liferay-frontend-projects/blob/master/projects/eslint-plugin/rules/general/lib/rules/no-anonymous-exports.js). Issue is [here](https://github.com/liferay/liferay-frontend-projects/issues/25)